test/kubetest/k8s.go

```diff
@@ -853,6 +853,7 @@ func (k8s *K8s) CreatePodsRaw(timeout time.Duration, failTest bool, templates ..
 	for _, podResult := range results {
 		if podResult == nil {
 			logrus.Errorf("Error - Pod should have been created, but is nil: %v", podResult)
+			errs = append(errs, errors.Errorf("Pod should have been created, but is nil: %v", podResult))
```

Comment:
We at least need pod name I think, since right now it is not clear why, probable we need to try to find some more details from Kubernetes here.